doc/source/tutorials/bipartite_matching/assets/maxflow2.py

```diff
@@ -0,0 +1,39 @@
+import igraph as ig
+import matplotlib.pyplot as plt
+
+g = ig.Graph(
+    9,
+    [(0, 4), (0, 5), (1, 4), (1, 6), (1, 7), (2, 5), (2, 7), (2, 8), (3, 6), (3, 7)],
+    directed=True
+)
+
+# Assign nodes 0-3 to one side, and the nodes 4-8 to the other side
+for i in range(4):
+    g.vs[i]["type"] = True
+for i in range(4, 9):
+    g.vs[i]["type"] = False
+
+# Add source and sink as nodes 9 and 10
+g.add_vertices(2)
+g.add_edges([(9, 0), (9, 1), (9, 2), (9, 3)]) # connect source to one side
+g.add_edges([(4, 10), (5, 10), (6, 10), (7, 10), (8, 10)]) # ... and sinks to the other
+
+flow = g.maxflow(9, 10)
+print("Maximal Matching is:", flow.value)
```

Comment:
Let's be precise: this is not "the matching". It is the _size_ of the matching. The matching consists of a set of edges. It is not a number.

Reply:
Fixed.